src/components/UI/docs/DocsNav.tsx

```diff
@@ -17,14 +17,22 @@ interface Props {
 }
 
 export const DocsNav: FC<Props> = ({ navLinks }) => {
+  const OPEN = 0;
+  const CLOSED = -1;
+  const [mobileAccordionState, setMobileAccordionState] = useState(CLOSED)
+
+  const updateMobileAccordionState = () => {
+    setMobileAccordionState(mobileAccordionState === OPEN ? CLOSED : OPEN)
+  }
```

Comment:
@corwintines I see what you're doing here so you can pass the variable as a `0` or `-1` to the below, but I think this makes things a bit tough to read. 

Could we maybe just use a boolean here, and label this a "toggle" function?

```suggestion
  const [isMobileAccordionOpen, setIsMobileAccordionOpen] = useState(false)

  const toggleMobileAccordion = () => {
    setMobileAccordionState(prev => !prev)
  }
```

...and then below use `index={isMobileAccordionOpen ? 0 : -1}`?

Reply:
(Feel free to push back, and remember if you do commit that, be sure to update the other variable name references)